src/kinetics/Kinetics.cpp

```diff
@@ -267,7 +269,7 @@ void Kinetics::checkReactionBalance(const Reaction& R)
     }
 }
 
-void Kinetics::selectPhase(const doublereal* data, const thermo_t* phase,
+void Kinetics::selectPhase(const doublereal* data, const ThermoPhase* phase,
```

Comment:
A chance here to change this to `double*` instead of `doublereal*`.

Reply:
I took the opportunity to update this in a few other places as well.